src/audio/pipewire/SDL_pipewire.c

```diff
@@ -519,7 +520,8 @@ get_int_param(const struct spa_pod *param, Uint32 key, int *val)
     prop = spa_pod_find_prop(param, NULL, key);
 
     if (prop && prop->value.type == SPA_TYPE_Int) {
-        spa_pod_get_int(&prop->value, &v);
+        if (spa_pod_get_int(&prop->value, &v) != 0)
+            return SDL_FALSE;
```

Comment:
This can be squashed into
```suggestion
    if (prop && spa_pod_get_int(&prop->value, &v) == 0) {
```

spa_pod_get_int() checks the value type and fails if it isn't SPA_TYPE_Int or the param is malformed (bad size), so these conditionals can be squashed into one.